### PR TITLE
Add Awesome-flexbox

### PR DESCRIPTION
**Pull request by willianjusten**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -54,6 +54,7 @@
 - [Web Performance Optimization](https://github.com/davidsonfellipe/awesome-wpo)
 - [Web Tools](https://github.com/lvwzhen/tools)
 - [Above-the-fold CSS Tools](https://github.com/addyosmani/above-the-fold-css-tools)
+- [Flexbox](https://github.com/afonsopacifer/awesome-flexbox)
```
Comment (sindresorhus) [inline readme.md]: Add it to the end.

**Comment by willianjusten** [comment]
Done.

Ps.: I was thinking here, why not to put in alphabetical order? I put the link there because I think was the best context...

**Comment by sindresorhus** [comment]
> Ps.: I was thinking here, why not to put in alphabetical order? I put the link there because I think was the best context...

https://github.com/sindresorhus/awesome/pull/166#issuecomment-106968913

**Comment by sindresorhus** [comment]
Thanks. Looks good. Would be cool if you could link back here in your list so users can easily discover more awesome lists ;)

**Comment by willianjusten** [comment]
I'll link back in my [awesome svg](https://github.com/willianjusten/awesome-svg) and I'll talk to my friend also.

Thanks!

**Pull request closed without merging**